src/main/java/podz/parser/Parser.java

```diff
@@ -5,6 +5,7 @@
 import podz.commands.DeadlineCommand;
 import podz.commands.DeleteCommand;
 import podz.commands.EventCommand;
+import podz.commands.FindCommand;
```

Comment:
Nice to put the find command as a seperate class. This makes the code more readable!